examples/oidc/config.yaml

```
@@ -0,0 +1,55 @@
+  oidc-issuer-url: "https://your-oidc-provider.example.com/realms/your-realm"
+  
+  # Client Configuration - replace with your client ID
+  client-id: "your-client-id"
+  
+  # OAuth2 Scopes
+  scope: "openid email profile"
+  
+  # Proxy Settings
+  http-address: "0.0.0.0:4180"
+  upstreams: "http://your-app:8080"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)